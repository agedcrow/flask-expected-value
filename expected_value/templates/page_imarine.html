{% extends "base.html" %}
{% block content %}

  <div class="p-4 text-center">
    <h2 class="fs-5 mt-2 mb-4">{{ title }}</h2>

    <p>Specs</p>
    <div class="mb-4 mx-auto" style="width:fit-content;">{{ tbl_specs | safe }}</div>

    <p>Rate table</p>
    <div class="mb-4 mx-auto" style="width:fit-content">{{ tbl_border | safe }}</div>
    
    <p>Result</p>
    <div class="mb-2 mx-auto" style="width:fit-content;">{{ tbl_result | safe }}</div>
    
    <div class="mb-4">
      <img src="data:image/png;base64,{{ plot_img }}" alt="Image", width="420"/>
    </div>

    <p>Machines</p>
    {% for d in machine_stats %}
    <div class="d-flex align-items-center justify-content-center">
      <div class="me-2">{{ d['machine_no'] }}</div>
      <div class="me-2">{{ d['count'] }}</div>
      <div>{{ d['mean'] }}</div>
    </div>
    {% endfor %}

  </div>

{% endblock%}