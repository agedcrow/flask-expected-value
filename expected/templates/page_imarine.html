{% extends "base.html" %}
{% block content %}

  <div class="p-4 text-center">
    <h2 class="fs-5 mt-2 mb-4">{{ title }}</h2>

    <p>Spec</p>
    <div class="mb-4 mx-auto" style="width:fit-content;">{{ tbl_spec | safe }}</div>

    <p>Rate</p>
    <div class="mb-4 mx-auto" style="width:fit-content">{{ tbl_border | safe }}</div>
    
    <p>Expected rate</p>
    <div class="mb-4 mx-auto" style="width:fit-content;">{{ tbl_expected | safe }}</div>
    
    <p>Real rate</p>
    <div class="mb-4 mx-auto" style="width:fit-content;">{{ tbl_real | safe }}</div>

    <div class="mb-4">
      <img src="data:image/png;base64,{{ plot_img }}" alt="Image", width="420"/>
    </div>

    <p>Machines</p>
    <div class="mb-4 mx-auto" style="width:fit-content;">{{ tbl_machine | safe }}</div>

  </div>

{% endblock%}